pandas/tests/arrays/interval/test_interval.py

```diff
@@ -443,3 +443,15 @@ def test_arrow_interval_type_error_and_warning():
     msg = "Argument `closed` is deprecated in favor of `inclusive`"
     with tm.assert_produces_warning(FutureWarning, match=msg, check_stacklevel=False):
         ArrowIntervalType(pa.int64(), closed="both")
+
+
+def test_interval_index_subtype():
+    # GH 46999
+    dates = date_range("2022", periods=3, tz="UTC")
+    result = IntervalIndex.from_arrays(
+        ["2022-01-01", "2022-01-02"],
+        ["2022-01-02", "2022-01-03"],
+        dtype="interval[datetime64[ns, UTC], both]",
```

Comment:
1. Could you parameterize over the `closed` options (I think there should be a fixture)
2. Could you also parameterize another timezone with UTC, e.g. `US/Pacific`
3. Will need a whatsnew note in v1.5

Reply:
Have added parameterize over the closed options, parameterize another timezone and whatsnew note in v1.5